### DISCUSSION TITLE
How to config the thread number for index construction in v2.3.x?

### DISCUSSION THREAD

**Discussion by Molly-Xu00** [milvus-io/milvus#31755] [Q&A and General discussion]
I am using Milvus 2.3.x standalone, how can i config the number of threads to create_index? If it does not support the user to config the threads, what is the default value of the thread number?

**Comment by xiaofan-luan** [comment]
@xiaocai2333 any ideas?

**Comment by Molly-Xu00** [comment]
Is the defualt number single thread or something like get_omp_num_procs()?

**Comment by xiaocai2333** [comment]
Before version 2.4, the number of threads used to build index was the number of cores.
After version 2.4, there is a configuration item to indicate the ratio of the number of indexing threads to the total number of cores.

**Comment by xiaocai2333** [comment]
The configuration is `common.buildIndexThreadPoolRatio` in milvus.yaml.

**Reply by Molly-Xu00** [reply]
I built Milvus standalone2.4,  but I didn't find milvus.yaml.  I just followed the instruction of install Milvus. The files after I start milvus are as follows.
<img width="193" alt="image" src="https://github.com/milvus-io/milvus/assets/65323198/b029153d-abf5-4077-8c57-c2cfb5a8eb0a">
